### ISSUE TITLE
Use Homey's location

### ISSUE THREAD

**Issue by Leliekevertje** [Inversion-NL/nl.inversion.wunderground#22]
I have inserted a manual location, see below

![image](https://cloud.githubusercontent.com/assets/19484735/20628443/40a879a4-b326-11e6-9bf1-fea09531a388.png)

I tested and saved this location. When I switch to antoher App (for instance Smart Presence) and go back to the Weather Underground App the checkbox ("vinkje" in dutch) spontaniously is set on (Use Homey's location.) See below

![image](https://cloud.githubusercontent.com/assets/19484735/20628500/998ba62c-b326-11e6-96bc-42f0717c1b4e.png)

Homey Insights shows the data based on Homey's location and not the manually set location (which I want).

Am I doing something wrong? Please help.

**Comment by Inversion-NL** [comment]
I'll check

**Comment by Inversion-NL** [comment]
Confirmed. Looks like a bug in Homey.
I'm correctly saving the variable.
Have to ask Athom.

**Comment by Inversion-NL** [comment]
Should be fixed in v0.3.0

**Comment by Leliekevertje** [comment]
Hi Inversion-NL,
Thanks for the good work! When do you expect v0.3.0 to go live?

**Comment by Inversion-NL** [comment]
As per request in issue https://github.com/Inversion-NL/nl.inversion.wunderground/issues/29

**Comment by gruijter** [comment]
Location is not updated correctly anymore from Homey (v1.3.0RC1) . I'm always placed in Den Ham by WU-app (v1.0.1). Homey has the correct location according set though. I have now set WU-app for manual location, but would really like to be able to use auto location :)

**Comment by Inversion-NL** [comment]
Is this still an issue?

**Comment by Leliekevertje** [comment]
Yes, it still is an issue (tested it today).

**Comment by leandresz** [comment]
Problem still remains, tested it a few seconds ago.

**Comment by Leliekevertje** [comment]
Hi Robin,

The bug is open for more than a year now, please tell me when you will fix it? (Use Homey's location is not an option for me.  When I do so, your app selects a weatherstation downtown The Hague which is at least 8 km away and it could be raining there while it's dry here). I really like to see this bug solved!

**Comment by WimHe** [comment]
Yes same here, please fix this bug and also provide the Rain rate so i can control luifels.
